artview/components/tools.py

```diff
@@ -105,6 +105,7 @@ def onPick(self, event):
         '''Get value at the point selected by mouse click.'''
         xdata = event.xdata # get event x location
         ydata = event.ydata # get event y location
+        #XXX this fails if not inside plot region
```

Comment:
Agreed, it's one I considered minor for now, but you're right we should take care of it.